### PR TITLE
Add DevFest NL

### PR DESCRIPTION
**Pull request by hvisser**

### CODE REVIEW

File: _conferences/devfest-nl-2016.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: "GDG DevFest"
+website: https://devfest.nl
+location: Amsterdam, The Netherlands
+
+date_start: 2016-10-08
+date_end:   2016-10-08
+
+cfp_start: 2016-08-01  # Optional
+cfp_end:   2016-17-09  # Optional
```
Comment (chiuki) [inline _conferences/devfest-nl-2016.md]: `2016-09-17`, perhaps?

Comment (hvisser) [reply _conferences/devfest-nl-2016.md]: Oops, US dates :)

Comment (JakeWharton) [reply _conferences/devfest-nl-2016.md]: This is ISO 8601! International standard!

Comment (hvisser) [reply _conferences/devfest-nl-2016.md]: I might have seen that format before somewhere...

**Comment by JakeWharton** [comment]
:+1:

**Pull request merged**